solana/program_v2_cli/.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
```

Comment:
I'd delete this `.github` folder... not sure, but we don't have it in any of our other projects

Reply:
@DiosAkhilleus

Reply:
done